code/game/jobs/job/civilian.dm

```diff
@@ -204,6 +204,36 @@
 	dufflebag = /obj/item/storage/backpack/duffel/eng
 	messengerbag = /obj/item/storage/backpack/messenger/engi
 
+/datum/job/officer/cargo
+	title = "Supply Officer"
+	flag = CARGOSEC
+	department = "Cargo"
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the quartermaster, head of personnel, and head of security"
```

Comment:
Having big issues with the quartermaster in that list

Reply:
Quartermasters are technically the ones in charge of cargo unless the head of personnel steps in. Other cargo staff are under their purview, so the officers should be as well.